cameras/paths.csv

```diff
@@ -41,6 +41,7 @@ Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}
 IC Realtime,nvr,NVR,(4C:11:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1;2,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,Stream {{stream}},Channel {{channel}},1,0,admin,admin,,http://icrealtime.com/docs/product-manual/Man_NVR716N.pdf
 IC Realtime,nvr,NVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://icrealtime.com/docs/product-manual/Man_DVRFLEX4E.pdf
 Luma,nvr,Lum-500/501 Series,(00:0F:0D),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}{{channel}}/Streaming/channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,main stream;sub stream,Channel {{channel}},01,02,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/Luma-500%20Technical%20Reference%20Manual.pdf
+Wirepath,camera,IP Camera,(00:0F:0D|D4:6A:91),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}},H.264,554,;v2,,Main Stream;Sub Stream,,,v2,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/WPS-750-BUL-IP-GR_Manual.pdf
```

Comment:
are 44 and 45 intentionally overlapping even though both are cameras with the same OUI?

Reply:
45 was preexisting but for whatever reason that RTSP url was not working for the user that I was supporting, only the new URL was working. I couldn't remember if we had an old user with the HTTP URL (line 45), so I left both assuming that the firmware would try both and choose the one that works.